src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -17,7 +17,7 @@
   <ItemGroup>
     <!-- This section is to force the version of non-direct dependencies -->
     <!-- the following package(s) are from https://github.com/dotnet/fxdac -->
-    <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
```

Comment:
Does this change need to be part of this PR?

Reply:
Nope, this clearly doesn't, but the CI builds all fail without that change....

Reply:
Other than this issue, the PR LGTM.

Reply:
OK, will let someone from the PS team chime in on what's going on here.

Reply:
Great.

Will fix the Code Factor issue, BTW.

Reply:
Thanks!

Reply:
> Will fix the Code Factor issue, BTW.

Done

Reply:
> OK, will let someone from the PS team chime in on what's going on here.

I've done this in a separate commit, JIC.

If this has to be done elsewhere (which would totally make sense) I'll probably need some help to exclude that commit from the PR, 'cos that's something I've never done.

Reply:
This change was made in https://github.com/PowerShell/PowerShell/pull/21084.

I would suggest updating the PR with a rebase.

Reply:
> I've done this in a separate commit, JIC.
> 
> If this has to be done elsewhere (which would totally make sense) I'll probably need some help to exclude that commit from the PR, 'cos that's something I've never done.

@sba923 Just revert that specific commit now that it's been in the PR mentioned above by @xtqqczze and ensure that you rebase from the powershell/main branch as that's cleaner than adding merge commits in your PR

Reply:
Hi, I have reverted the commit (which results in the CI build to now fail with `/home/runner/work/PowerShell/PowerShell/src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj : error NU1903: Warning As Error: Package 'System.Data.SqlClient' 4.8.5 has a known high severity vulnerability, https://github.com/advisories/GHSA-98g6-xh36-x2p7 [/home/runner/work/PowerShell/PowerShell/src/powershell-unix/powershell-unix.csproj]`), but I can't seem to get the rebase to work. Can you please help me out with that (either using the git CLI or VScode)?